rasterio/rio/info.py

```diff
@@ -50,13 +55,22 @@
 @click.option('--checksum', 'meta_member', flag_value='checksum',
               help="Print integer checksum of a single band "
                    "(use --bidx).")
+@click.option('--gcps', 'meta_member', flag_value='gcps',
+              help="Print JSON representation of ground control points.")
```

Comment:
Is there any chance that we'd want a command for writing gcps? If so, maybe we could have a separate `rio gcp` command for reading/writing GCPS (following the lead of  `rio overview`).